etl/pypasar/omop/visit_occurrence.py

```diff
@@ -22,15 +22,48 @@ def execute(self):
             raise err
 
     def initialize(self):
-        pass
+        with self.engine.connect() as connection:
+            with connection.begin():
+                # Set the schema for subsequent SQL operations
+                connection.execute(
+                    text(f'SET search_path TO {os.getenv("POSTGRES_OMOP_SCHEMA")}'))
+                # Drop view along with its dependent objects
+                connection.execute(text("DROP VIEW IF EXISTS stg__visit_occurrence CASCADE"))
+                connection.execute(text("DROP VIEW IF EXISTS int__visit_occurrence CASCADE"))
+                # Clear all existing rows from the visit_occurrence table
+                connection.execute(text("TRUNCATE TABLE visit_occurrence"))
 
     def process(self):
-        # In batches
-        # Read from source
-        # Transform
-        # Ingest into OMOP Table
-        pass
+        # List of SQL file paths
+        sql_files = [
+            os.path.join(os.getenv("BASE_PATH"), "visit_occurrence/stg__visit_occurrence.sql"),
+            os.path.join(os.getenv("BASE_PATH"), "visit_occurrence/int__visit_occurrence.sql"),
+            os.path.join(os.getenv("BASE_PATH"), "visit_occurrence/visit_occurrence.sql")
```

Comment:
To pass folder and file as arguments for consistency, ex: `os.path.join(os.getenv("BASE_PATH"), "visit_occurrence", "visit_occurrence.sql")`